### PR TITLE
Add support for PowerPC architecture

### PR DESCRIPTION
**Pull request by mterron**
Related to https://github.com/pi-hole/FTL/pull/88

**By submitting this pull request, I confirm the following (please check boxes, eg [X]) _Failure to fill the template will close your PR_:**

***Please submit all pull requests against the `development` branch. Failure to do so will delay or deny your request***

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] I have checked that [another pull request](https://github.com/pi-hole/pi-hole/pulls) for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:**

1

---
Add support for PowerPC pihole-FTL

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

### CODE REVIEW

**Comment by Mcat12** [comment]
>  I have read and understood the contributors guide.

For the future:
> Submit Pull Requests to the development branch only.

**Comment by dschaper** [comment]
Since were in a Feature Freeze for the next release, I've tagged this to the next milestone.

File: automated install/basic-install.sh [hunk 1]
```diff
@@ -1265,6 +1265,10 @@ FTLdetect() {
       echo ":::  Detected ARM architecture"
       binary="pihole-FTL-arm-linux-gnueabi"
     fi
+  elif [[ $machine == ppc ]]; then
```
Comment (Mcat12) [inline automated install/basic-install.sh]: Are there any other names PowerPC might go by?

**Comment by mterron** [comment]
ppc is the only one

File: automated install/basic-install.sh (hunk 1 above)
Comment (mterron) [inline automated install/basic-install.sh]: ppc is the only one I'm aware of.

**Pull request merged**